pypowsybl/network.py

```diff
@@ -3018,6 +3027,14 @@ def create_eurostag_tutorial_example1_with_power_limits_network() -> Network:
     """
     return _create_network('eurostag_tutorial_example1_with_power_limits')
 
+def create_eurostag_tutorial_example1_with_apc_extension() -> Network:
```

Comment:
Probably better to hide more this method, for example in the unit test file, since it's really more targetted at unit testing than as a case for users.

Reply:
It looks good this way, thanks :)